src/index.ts

```diff
@@ -60,7 +60,7 @@ const options = {
 
 Database.loadDb()
 
-const cakebot = new Client(
+export const cakebot = new Client(
```

Comment:
This should not directly be exported, but instead be proxied through commands like what `commands/internal/pyramid/arrest` and its friends do with the 'admins' field.